actions/utils.py

```diff
@@ -12,19 +14,32 @@ def create_action(user, verb,  activity_json, target=None):
                                             created__gte=last_minute)
     if target:
         target_ct = ContentType.objects.get_for_model(target)
-        similar_actions = similar_actions.filter(target_ct=target_ct,
+        print('target_ct',target_ct)
+        if 'article' in str(target_ct) :
+            print('if')
```

Comment:
Printleri mastera commit etmeyelim @alper-the-last

Reply:
Printler çıkartıldı